AirRobeWidget/src/main/kotlin/com/airrobe/widgetsdk/airrobewidget/widgets/AirRobeConfirmation.kt

```diff
@@ -40,6 +41,8 @@ class AirRobeConfirmation @JvmOverloads constructor(
 
     private var orderId: String? = null
     private var email: String? = null
+    private var orderSubTotalCents: Int = AirRobeConstants.INT_NULL_MAGIC_VALUE
```

Comment:
"SubTotal" is one word, so should be "orderSubtotalCents" like in the docs below
https://connector.airrobe.com/docs/android